### DISCUSSION TITLE
How would replace the "no extention" sidebar with one that I use with Sideberry?

### DISCUSSION THREAD

**Discussion by YamDotPNG** [soulhotel/FF-ULTIMA#11] [Q&A]
does anyone know how?

**Comment by soulhotel** [comment]
Someone else also wanted this a day or two ago, so I added a file to the [replace features list](https://github.com/soulhotel/FF-ULTIMA/blob/main/replace%20features/%23replace-replace-features.md#-chromethemesposition-verttabcss). Simply download this .css file, and replace it with your original. There is a live preview gif of what it looks like.

However note, I styled the positioning of the sidebar header, BUT this isnt a default feature of the theme. So you will have to handle centering things such as the sideberys tab icons, and adding margins to the tab title text, etc. Luckily sidebery makes that very easy to do.
